pynsee/macrodata/get_series.py

```diff
@@ -131,28 +130,38 @@ def get_series(
 
         list_data.append(df)
 
-    data = pandas.concat(list_data).reset_index(drop=True)

```

Comment:
please remove all following changes from this PR

Reply:
I think these changes come from "conflicts" between the last release and this branch's original source.

Reply:
OK, then let's update the code to reflect the new release: please readd ``data = pandas.concat(list_data).reset_index(drop=True)``